File: octo/oswatchRefactor/src/main/scala/com/sankuai/octo/oswatch/model/OctoEnv.scala
package com.sankuai.octo.oswatch.model

/**
 * Created by dreamblossom on 15/9/30.
 */
object OctoEnv extends Enumeration {
  val test = "test"
  val stage = "stage"
  val prod = "prod"

  def getEnv(v: Int) = {
    v match {
      case 1 => test
      case 2 => stage
      case _ => prod
    }
  }
}

File: octo/oswatch/src/main/java/com/sankuai/octo/oswatch/model/OctoEnv.java
package com.sankuai.octo.oswatch.model;

/**
 * Created by chenxi on 8/12/15.
 */
public class OctoEnv {
    public static String getEnv(int value) {
        switch (value) {
            case 1: return "test";
            case 2: return "stage";
            case 3: return "prod";
            default: return "prod";
        }
    }
}
